FeaturePoints/src/cz/fidentis/featurepoints/FeaturePointsUniverse.java

```diff
@@ -523,5 +523,31 @@ public void resetSimplify(){
     public List<FacialPoint> getAllPoints() {
         return facialPoints;
 }
+    
+    //PDM methods
+    
+    public double[] calculateCurvature(CurvatureType curveType, boolean f) {
+        curveControl = new CurvatureControl(boundaryVertices, elementSet);
+        return curveControl.computeCurvature(curveType);
+    }
+    
+    public CornerTable getCornerTable() {
+        return this.cornerTable;
+    }
+    
+    public Set<Set<Integer>> findNose(boolean notUsed) {  
+        calculateAnisotropicDenoise(10, PwSmooth.METHOD_ANISOTROPIC, 2.0, true);
+        calculateCurvature(CurvatureType.Minimum);
+        thresArea = new ThresholdArea(elementSet, cornerTable, boundaryVertices);
+        calculateThresholdFaces(15);
+        return findNoseTipArea(notUsed);
+    }
+    
+    public Set<Set<Integer>> findNoseTipArea(boolean notUsed) {
+        return thresArea.findNoseTip2(thresholdFaces, notUsed);

```

Comment:
to bol len docasny nazov, premenuj to prosimta na findAllNoseTipAreas alebo nieco podobne. Tiez samotnu metodu bude treba refaktorovat, kvoli duplikacii kodu